docs/resources/extra_disk_size.md

```diff
@@ -29,21 +29,28 @@ resource "cloudamqp_instance" "instance" {
   rmq_version = "3.10.1"
 }
 
-data "cloudamqp_nodes" "nodes" {
+# Resize disk with 25 extra GB
+resource "cloudamqp_extra_disk_size" "resize_disk" {
   instance_id = cloudamqp_instance.instance.id
+  extra_disk_size = 25
+  depends_on = [
+    cloudamqp_instance.instance,
```

Comment:
Is it necessary? Should already have dependency to cloudamqp_instance.instance due to `instance_id = cloudamqp_instance.instance.id`

Reply:
Just tried out this configuration while trying out configurable sleep. Resize kicks in once the instance is created and gets the dependency to `cloudamqp_instance.instance` set in the state file. Once all is finished I got a new instance with the extra disk.

```
resource "cloudamqp_instance" "instance" {
  name   = "Instance"
  plan   = "squirrel-1"
  region = "amazon-web-services::us-west-2"
  tags = []
  rmq_version = "3.10.5"
}

resource "cloudamqp_extra_disk_size" "resize_disk" {
  instance_id = cloudamqp_instance.instance.id
  extra_disk_size = 5
}
```